src/icon/line/BoxAi.js.flow

```diff
@@ -0,0 +1,30 @@
+// @flow
+/* eslint-disable react/jsx-sort-props */
+import * as React from 'react';
+import * as vars from '../../styles/variables';
+import AccessibleSVG from '../../icons/accessible-svg';
+import type { Icon } from '../../icons/flowTypes';
+
+/**
+ * This is an auto-generated component and should not be edited
+ * manually in contributor pull requests.
+ *
+ * If you have problems with this component:
+ * - https://github.com/box/box-ui-elements/issues/new?template=Bug_report.md
+ *
+ * If there are missing features in this component:
+ * - https://github.com/box/box-ui-elements/issues/new?template=Feature_request.md
+ */
+
+const BoxAi = (props: Icon) => (
```

Comment:
Should the title/file name be BoxAI?

Reply:
this is an autogenerated name based on the asset file in the design-assets / blueprint-assets internal repo